### DISCUSSION TITLE
Mobs don't open doors

### DISCUSSION THREAD

**Discussion by metallnt** [TownyAdvanced/Towny#6735] [Q&A] [answered]
I disabled plugins one by one. Just using towny noticed this. How can I get them back home? (close doors successfully)

**Comment by LlmDl** [comment, marked as answer]
You would want to do something like remove wooden doors from the config switch_ids list, then use wooden doors.

**Reply by metallnt** [reply]
But will it help with the action of mobs?
I will try to do it and write.

**Reply by metallnt** [reply]
tomorrow

**Reply by metallnt** [reply]
It's work. Thanks
